### PR TITLE
MH-254: Implemented Alert MUI Components to Success/Error messages

### PR DESCRIPTION
**Pull request by sunzae99**

### CODE REVIEW

**Review by aaw0151 — APPROVED** [review]
Looks good to me!

**Review by mattfredericksen — APPROVED** [review]
@sunzae99 checks are successful here, so you should be able to merge this after merging `main` into this branch.

**Pull request merged**
